tests/clar_libgit2.c

```diff
@@ -558,5 +558,36 @@ bool cl_sandbox_supports_8dot3(void)
 
 	return supported;
 }
+
+bool cl_sandbox_supports_reparse_points(void)
+{
+	unsigned int flags = 0;
+	char path[8];
+	char drive = 'A' + _getdrive() - 1;
+
+	sprintf(path, "%c:\\", drive);
+
+	GetVolumeInformation(path, NULL, 0, NULL, NULL, &flags, NULL, 0);
```

Comment:
Agreed. You (@ethomson) and I had discussed something similar to this previously. It completely slipped my mind when I prepared this PR. I'll get it fixed up asap. Thanks.